workbench/static/workbench/js/runtime/logger.js

```diff
@@ -37,7 +40,7 @@ XBlock generated them and other XBlock-local metadata.
             sendRequest, has;
 
         sendRequest = function(data, options) {
-	    console.log(data);
+	    console.log(JSON.stringify(data));
```

Comment:
@macdiesel You don't _need_ to but each outputs slightly differently.

The latter prints a string, while the former prints a dictionary. The distinction here is that `console.log(JSON.stringify(data));` will condense the object/dictionary into a single line string which is output. On the other hand, the original `console.log(data);` will output the value as in "interactive" object; you can click arrows to selectively show/hide keys/values.

I find this (the interactive form, `console.log(data);`) to be particularly useful for large dictionaries with many keys/values. 

`stringify`ing can be useful to see a small number of keys/values in a condensed, one-line format.